packages/nivo-calendar/src/enhance.js

```diff
@@ -24,7 +25,7 @@ export default Component =>
         withPropsOnChange(['data', 'domain', 'colors'], ({ data, domain, colors }) => {
             let colorDomain
             if (domain === 'auto') {
-                colorDomain = [minBy(data, 'value').value, maxBy(data, 'value').value]
+                colorDomain = [get(minBy(data, 'value'), 'value'), get(maxBy(data, 'value'), 'value')]
```

Comment:
a more simple way to handle this could be:

```javascript
if (data.length === 0) {
    colorDomain = [0, 0] 
}
```

This way, we do not have to call four functions when we know we'll get nothing, what do you think?